python/gui/auto_generated/qgsexpressiontreeview.sip.in

```diff
@@ -0,0 +1,258 @@
+/************************************************************************
+ * This file has been generated automatically from                      *
+ *                                                                      *
+ * src/gui/qgsexpressiontreeview.h                                      *
+ *                                                                      *
+ * Do not edit manually ! Edit header and run scripts/sipify.pl again   *
+ ************************************************************************/
+
+
+
+
+
+
+
+
+class QgsExpressionItem : QStandardItem
```

Comment:
We should to-do remove this from sip in 4.0

Reply:
it's emitted in a signal from the tree (current changed)